src/services/completions.ts

```diff
@@ -371,6 +381,8 @@ namespace ts.Completions {
                 }
             }
 
+            result = result.filter(entry => entry.name.indexOf(baseName) >= 0);

```

Comment:
Is this deliberately fuzzy, rather than just "starts with"? i.e. if the user had typed "./dir/ab|" and was getting completions at the "|", then "xxab" would match the filter as well as "abc".